### PR TITLE
{plugins,src}/: Fix typo ('overridden' is spelled with double 'd').

### PR DESCRIPTION
**Pull request by sunweaver**
This also affects the ABI/API to a considerable extent.
 .
 However, before uploading this package to Debian, I'd prefer to
 see this amended.

### CODE REVIEW

File: src/plugin/updatenotifier.h [hunk 1]
```diff
@@ -58,12 +58,12 @@ class UpdateNotifier
...
 
     void notify(MImUpdateEvent *event);
-    void notifyOverride(const Logic::KeyOverrides &overriden_keys,
+    void notifyOverride(const Logic::KeyOverrides &overridden_keys,
                         bool update = false);
 
     Q_SIGNAL void cursorPositionChanged(int cursor_position,
                                         const QString &surrounding_text);
-    Q_SIGNAL void keysOverriden(const Logic::KeyOverrides &overriden_keys,
+    Q_SIGNAL void keysOverriden(const Logic::KeyOverrides &overridden_keys,
```
Comment (dobey) [inline src/plugin/updatenotifier.h:66]: Shouldn't you also fix the signal name, and the connections where it's used, rather than just variable names? :)

**Review by dobey — CHANGES_REQUESTED** [review]

**Comment by dobey** [comment]
> This also affects the ABI/API to a considerable extent.

Considering it is all private API though, it shouldn't be an issue, as different versions of maliit-keyboard things shouldn't be mixed.

File: src/plugin/updatenotifier.h (hunk 1 above)
Comment (sunweaver) [reply src/plugin/updatenotifier.h]: I'll update the patch in Debian's upload, as well.

**Review by dobey — APPROVED** [review]
Looks good now, thanks.

**Pull request merged**